src/SDKs/Batch/DataPlane/Tools/ConfigureAwaitAnalyzer/ConfigureAwaitAnalyzer.Tests/ConfigureAwaitAnalyzer.Tests.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!--<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />-->
+  <PropertyGroup>
```

Comment:
@matthch you do not need to add all these properties in the project file? Any reason you are adding all these properties?